File: 251001/Korotkaya/Publisher/Entity/Editor.cs
namespace WebApplication1.Entity
{
    public class Editor
    {
        public long Id { get; set; }
        public string Login { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;
        public List<Topic> Topics { get; set; } = new();
    }
}

File: 251002/Demidovets/laba2/models/editor.py
from sqlalchemy.orm import Mapped, mapped_column, relationship
from sqlalchemy import Text
from .base import Base


class Editor(Base):
    __tablename__ = 'tbl_editor'

    login: Mapped[str] = mapped_column(Text, nullable = False, unique = True)
    password: Mapped[str] = mapped_column(Text, nullable = False)
    firstname: Mapped[str] = mapped_column(Text, nullable = False)
    lastname: Mapped[str] = mapped_column(Text, nullable = False)
   
    # Связь один ко многим с таблицей News
    news = relationship("News", back_populates="editor")